downstream/modules/devtools/proc-rhdh-create.adoc

```diff
@@ -0,0 +1,76 @@
+. Navigate to the *Overview* page.
+. Click *Create*.
+. Click *Create Ansible Git Project*.
+. The *Available Templates* page opens.
+. Click *Create Ansible Playbook project*.
+. In the *Create Ansible Playbook Project* page, enter information for your new project in the form.
++
+You can see sample values for this form in the Example project.
++
+[options="header"]
+|===
+|Field |Description
+|Source code repository organization name or username
+|The name of your source code repository username or organization name
+|Playbook repository name 
+|The name of your new Git repository
+|Playbook description
+(Optional)
+|A description of the new playbook project
+|Playbook project's collection namespace
+|The new playbook Git project creates an example collection folder for you. 
+Enter a value for the collection namespace.
+|Playbook project's collection name
+|The name of the collection
+|Catalog Owner Name
+|The name of the Developer Hub catalog item owner.
+This is a Red Hat Developer Hub field.
+|Source code repository organization name or username
+|The name of your source code repository username or organization name
+|Playbook repository name
+|The name of your new Git repository
+|Playbook description (Optional)
+|A description of the new playbook project
+|System (Optional)
+|This is a {RHDH} field
+|===
+. Click *Review*.
+
+
+
+To view the dashboard for Ansible plug-ins for {RHDH}, click *Ansible* in the {RHDH} navigation panel.
+
+image::rhdh-plugin-dashboard.png[Ansible plug-in dashboard]
+
+The plug-in dashboard illustrates the steps you need to take from learning about Ansible to deploying automation jobs from {PlatformNameShort}:
+
+* *Overview* displays the main dashboard page. 
+* *Learn* provides links to resources curated by Red Hat that introduce you to Ansible and provide step-by-step examples to get you started.
+For more information, see
+xref:rhdh-learning_rhdh-using[Learning about Ansible].
+* *Discover existing collections* links to {PrivateHubName}, if configured in the plug-ins, or to {HubName} hosted on the Red Hat Hybrid Cloud Console.
+{HubNameStart} stores existing collections and execution environments that you can use in your projects.
+For more information, see
+xref:rhdh-discover-collections_rhdh-using[Discovering existing Ansible content].
+* *Create* creates new projects in your configured Source Control Management platforms such as GitHub.
+For more information, see the Creating a project section.
+* *Develop* links you to OpenShift Dev Spaces, if configured in the Ansible plug-ins installation.
+OpenShift Dev Spaces provides on-demand, web-based Integrated Development Environments (IDEs), where you can develop automation content.
+For more information, see the Developing projects section.
+* *Operate* connects you to {PlatformNameShort}, where you can create and run automation jobs that use the projects you have developed. 
+For more information, see the Operating automation tasks section.
```

Comment:
May have missed this but is there an xref for the **Operating automation tasks** section?